crates/core/src/db/queries/metadatas.rs

```diff
@@ -89,7 +92,9 @@ pub fn list(
                 metadata_jsons::image,
             ))
             .distinct()
-            .order(metadatas::address.asc())
+            .order((listing_receipts::price.desc(), metadatas::name.asc()))
```

Comment:
How does this perform at scale?  We've had problems with this particular line before.

Reply:
```
Indexes:
    "listing_receipts_pkey" PRIMARY KEY, btree (address)
    "listing_receipts_address_index" btree (address)
    "listing_receipts_auction_house_index" btree (auction_house)
    "listing_receipts_metadata_index" btree (metadata)
    "listing_receipts_purchase_receipt_idx" btree (purchase_receipt)
```

```
Indexes:
    "metadatas_pkey" PRIMARY KEY, btree (address)
    "meta_name" btree (name)
    "metadatas_mint_address_index" hash (mint_address)
```

We have index on metadata name. Definitely need one on lr price. can you add a migration? just reminder when adding migrations make sure to have db url set to your local db.

still need to do performance comparison but index needed to give it a shot.